packages/nvidia_nat_core/src/nat/front_ends/fastapi/message_handler.py

```
@@ -250,13 +250,15 @@ def _done_callback(_task: asyncio.Task):
                     if self._conversation_id:
                         self._worker.remove_conversation_handler(self._conversation_id)
 
+                if self._workflow_schema_type is None:
+                    raise ValueError("Workflow schema type is not initialized")
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)